Rebooted the test to be sure, and Travis still doesn't like it:
```bash
In test/lint/lint-includes.sh line 15:
cd "$(dirname $0)/../.." > /dev/null
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
In test/lint/lint-includes.sh line 114:
cd - > /dev/null
^--------------^ SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
^--------------^ SC2103: Use a ( subshell ) to avoid having to cd back.
For more information:
  https://www.shellcheck.net/wiki/SC2164 -- Use 'cd ... || exit' or 'cd ... |...
  https://www.shellcheck.net/wiki/SC2103 -- Use a ( subshell ) to avoid havin...
^---- failure generated from test/lint/lint-shell.sh
```

--- a/test/lint/lint-includes.sh
+++ b/test/lint/lint-includes.sh
@@ -11,6 +11,9 @@
 export LC_ALL=C
 IGNORE_REGEXP="/(leveldb|secp256k1|univalue)/"
 
+# cd to root folder of git repo for git ls-files to work properly
+cd "$(dirname $0)/../.." || exit 1
+
 filter_suffix() {
     git ls-files | grep -E "^src/.*\.${1}"'$' | grep -Ev "${IGNORE_REGEXP}"
 }

